apps/api/v2/src/ee/calendars/controllers/calendars.controller.e2e-spec.ts

```diff
@@ -192,13 +192,19 @@ describe("Platform Calendars Endpoints", () => {
       .expect(200);
   });
 
-  it.skip(`/POST/v2/calendars/${OFFICE_365_CALENDAR}/disconnect: it should respond with a 201 returning back the user deleted calendar credentials`, async () => {
+  it(`/POST/v2/calendars/${GOOGLE_CALENDAR}/disconnect: it should respond with a 201 returning back the user deleted calendar credentials`, async () => {
```

Comment:
@ThyMinimalDev the tests should work now since we're using the google calendar credentials now!